app/controllers/AdditionalMetadataController.scala

```diff
@@ -17,28 +20,66 @@ class AdditionalMetadataController @Inject() (
 
   def start(consignmentId: UUID): Action[AnyContent] = standardTypeAction(consignmentId) { implicit request: Request[AnyContent] =>
+    getStartPageDetails(consignmentId, request.token)
+      .map(pageArgs => Ok(views.html.standard.additionalMetadataStart(pageArgs)))
+      .recover(err => {
+        logger.error(err.getMessage, err)
+        Forbidden(views.html.forbiddenError(request.token.name, isLoggedIn = true, isJudgmentUser = false))
+      })
+  }
+
+  private def getStartPageDetails(consignmentId: UUID, token: Token) = {
     for {
-      consignment <- consignmentService.getConsignmentDetails(consignmentId, request.token.bearerAccessToken)
+      consignment <- consignmentService.getConsignmentDetails(consignmentId, token.bearerAccessToken)
       (closurePropertiesSummaries, descriptivePropertiesSummaries) <- displayPropertiesService
-        .getDisplayProperties(consignmentId, request.token.bearerAccessToken, None)
+        .getDisplayProperties(consignmentId, token.bearerAccessToken, None)
         .map(_.partition(byClosureType))
         .map(m => (m._1.map(_.summary), m._2.map(_.summary)))
     } yield {
-      val pageArgs = AdditionalMetadataStartPage(
+      AdditionalMetadataStartPage(
         consignment.consignmentReference,
         consignmentId,
-        request.token.name,
+        token.name,
         closurePropertiesSummaries,
         descriptivePropertiesSummaries,
         getValue(consignment.consignmentStatuses, ClosureMetadataType),
         getValue(consignment.consignmentStatuses, DescriptiveMetadataType)
       )
-      Ok(views.html.standard.additionalMetadataStart(pageArgs))
+    }
+  }
+
+  def validate(consignmentId: UUID): Action[AnyContent] = standardTypeAction(consignmentId) { implicit request: Request[AnyContent] =>
+    (for {
+      pageArgs <- getStartPageDetails(consignmentId, request.token)
+      statuses <- consignmentService.getConsignmentFilesData(consignmentId, request.token.bearerAccessToken)
+    } yield {
+      val allErrors = getErrors(statuses, "descriptive", DescriptiveMetadataType) ++
+        getErrors(statuses, "closure", ClosureMetadataType)
+      if (allErrors.nonEmpty) {
+        val args = pageArgs.copy(errors = allErrors)
+        Ok(views.html.standard.additionalMetadataStart(args))
+      } else {
+        Redirect(routes.DownloadMetadataController.downloadMetadataPage(consignmentId))
+      }
+    }).recover(err => {
+      logger.error(err.getMessage, err)
+      Forbidden(views.html.forbiddenError(request.token.name, isLoggedIn = true, isJudgmentUser = false))
+    })
+  }
+
+  private def getErrors(statuses: GetConsignmentFiles.getConsignmentFiles.GetConsignment, metadataType: String, statusType: StatusType): Seq[(String, List[String])] = {
+    val incompleteCount = statuses.files.flatMap(_.fileStatuses).filter(_.statusType == statusType.id).count(_.statusValue == "Incomplete")
```

Comment:
Could you use the case object value type instead of hardcoding "Incomplete" value, ie `IncompleteValue.value`